test/DynamoCoreTests/Updates/UpdatesTests.cs

```diff
@@ -47,5 +51,88 @@ public void UpdateManagerConfiguration_LoadTest()
             Assert.IsNull(UpdateManagerConfiguration.Load("filePath", updateManager));
             Assert.IsNull(UpdateManagerConfiguration.Load(Path.GetTempFileName(), updateManager));
         }
+
+        [Test]
+        [Category("UnitTests")]
+        public void UpdateManagerConfiguration_SaveTest()
+        {
+            var config = new UpdateManagerConfiguration();
+            Assert.DoesNotThrow(() => config.Save(Path.GetTempFileName(), updateManager));
+        }
+
+        [Test]
+        [Category("UnitTests")]
+        public void CheckNewerDailyBuildTest()
+        {
+            var config = new UpdateManagerConfiguration();
+            var oldValue = config.CheckNewerDailyBuild;
+
+            config.CheckNewerDailyBuild = !config.CheckNewerDailyBuild;
+            Assert.AreEqual(!oldValue, config.CheckNewerDailyBuild);
+        }
+
+        [Test]
+        [Category("UnitTests")]
+        public void ForceUpdateTest()
+        {
+            var config = new UpdateManagerConfiguration();
+            var oldValue = config.ForceUpdate;
+
+            config.ForceUpdate = !config.ForceUpdate;
+            Assert.AreEqual(!oldValue, config.ForceUpdate);
+        }
+        #endregion
+
+        #region UpdateManager
+        [Test]
+        [Category("UnitTests")]
+        public void QuitAndInstallUpdateTest()
+        {
+            Assert.DoesNotThrow(() => updateManager.QuitAndInstallUpdate());
+        }
+
+        [Test]
+        [Category("UnitTests")]
+        public void HostApplicationBeginQuitTest()
+        {
+            Assert.DoesNotThrow(() => updateManager.HostApplicationBeginQuit());
+        }
+        #endregion
+
+        #region DynamoLookup
+        private class DynamoLookupChildTest : DynamoLookUp
+        {
+            public override IEnumerable<string> GetDynamoInstallLocations()
+            {
+                return new List<string>();
+            }
+        }
+
+        [Test]
+        [Category("UnitTests")]
+        public void DynamoLookupTests()
+        {
+            var lookup = new DynamoLookupChildTest();
+
+            Assert.IsNull(lookup.LatestProduct);
```

Comment:
Can you explain or comment here why it would be null?

Reply:
Added a comment regarding this. It is related to the implementation of GetDynamoInstallLocations in my test class.